app/helpers/cases_helper.rb

```diff
@@ -276,14 +276,14 @@ def request_details_html(kase)
     content_tag(:strong, "#{kase.subject} ", class: 'strong') +
```

Comment:
I think there is some refactoring of this method that could be done as we check `third_party?` a lot - have spoken to @njpreddy about this and we said we'd pair on it after the migration 👍